molgenis/molgenis-emx2
# chore(directory): updates MIABIS core V3

how to test:
- go to the preview: and check Schema (ontology) tables. 
- Please note that the Directory App is broken!

todo:
- [ ] fix Directory app
